src/averageAges.js

```diff
@@ -15,11 +15,32 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
```

Comment:
replace `if ()` statement with &&, || or ?:

looks like a future reject. Your code contain a lot of "if"